public/index.html

```diff
@@ -1,47 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <script id="Cookiebot" src="https://consent.cookiebot.com/uc.js" data-cbid="54349014-3934-489b-a905-584bc83272b2" data-blockingmode="auto" type="text/javascript"></script>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <script src="https://www.gstatic.com/firebasejs/7.17.1/firebase-app.js"></script>
-    <script
-      defer
-      src="https://www.gstatic.com/firebasejs/7.17.1/firebase-analytics.js"
-    ></script>
-    <script
-      defer
-      src="https://www.gstatic.com/firebasejs/7.17.1/firebase-firestore.js"
-    ></script>
-    <script
-      src="https://kit.fontawesome.com/0d4aacb00e.js"
-      crossorigin="anonymous"
-    ></script>
-    <title>MyCovidConnect: Find Nearest Healthcare facility</title>
-  </head>
+	<head>
+		<script
+			id="Cookiebot"
+			src="%REACT_APP_COOKIEBOT_CONSENT_HEAD%"
```

Comment:
`COOKIEBOT_CONSENT_HEAD` that is how you should be declaring and accessing env vars; change anywhere else likewise;